source/content/new-relic.md

```diff
@@ -219,6 +220,8 @@ Yes! If you have a [Supported Site Plan](#supported-site-plans), you can bring y
 * Your New Relic [License Key](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key#finding)
 * Your New Relic [Account ID](https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id#finding)
 
+**Note:** Pantheon SSO cannot be used to sign in to New Relic for sites using their own license. You will not be able to sign in to New Relic directly from the Pantheon Dashboard.
```

Comment:
@ari-gold, I added the second sentence to clarify @alexfornuto's changes. Can you give us a 👍  or 👎  on this, please?